Patches/Qt/itemviews.cpp

```diff
@@ -393,7 +393,7 @@ bool QAccessibleTable2::unselectColumn(int column)
     QModelIndex index = view()->model()->index(0, column, view()->rootIndex());
     if (!index.isValid() || view()->selectionMode() & QAbstractItemView::NoSelection)
         return false;
-    view()->selectionModel()->select(index, QItemSelectionModel::Columns & QItemSelectionModel::Deselect);
+    view()->selectionModel()->select(index, static_cast<QItemSelectionModel::SelectionFlags> (QItemSelectionModel::Columns & QItemSelectionModel::Deselect));
```

Comment:
Whee! This is actually a bug in Qt; the `&` should be a `|`. (This change is just hiding the actual bug.)

Interestingly, ViViA had an instance of this exact same bug :smile:.